edb/pgsql/metaschema.py

```diff
@@ -5952,7 +5960,9 @@ def construct_pg_view(table_name: str, columns: List[str]) -> dbops.View:
             ),
             returns=('bool',),
             text="""
-                SELECT has_schema_privilege(schema_oid, privilege)
+                SELECT COALESCE(
```

Comment:
Do you need both this and the above UNION trick?

Reply:
These are separate functions because they have different signatures.

I wanted to avoid the UNION trick, but couldn't figure out how.

Reply:
Oh, I can just use COALESCE whose first arg is a sublink.

Note: SQL's behavior when converting a result of a relational query to a scalar that can be coalesced and returned from a function is not intuitive.

Reply:
Also, I'll be squashing this fix into a commit, because I intend to "rebase and merge"